dpdusage.ml

```diff
@@ -12,9 +12,44 @@ module Node = Dpd_compute.Node
 
 let version_option = ref false
 let print_path_option = ref true
+let strict_whitelist = ref false
 
 let threshold_option = ref 0
 
+module Ident_set = Set.Make(String)
+
+let whitelisted = ref Ident_set.empty
+
+let add_whitelisted_item s =
+  whitelisted := Ident_set.add s !whitelisted
+
+let remove_surrounding_white_space s =
```

Comment:
Would be better to use [`String.trim`](http://caml.inria.fr/pub/docs/manual-ocaml/libref/String.html#VALtrim).